build/ChangeVersion.proj

```diff
@@ -28,12 +28,20 @@
       <ExpectedMatchCount>1</ExpectedMatchCount>
     </AssemblyInformationalVersion>
 
-    <VsixVersion Include="$(SolutionRoot)\src\Integration.Vsix\Manifests\VS201*\source.extension.vsixmanifest">
+    <!-- Set VSIX version in all VSIXes, including the embedded vsix -->
+    <VsixVersion Include="$(SolutionRoot)\src\Integration.Vsix\Manifests\VS201*\source.extension.vsixmanifest;$(SolutionRoot)\src\EmbeddedVsix\source.extension.vsixmanifest">
       <Find>(?&lt;=Identity.*Version=")([^"]*)</Find>
       <ReplaceWith>$(VsixVersion)</ReplaceWith>
       <ExpectedMatchCount>1</ExpectedMatchCount>
     </VsixVersion>
 
+    <RefToEmbeddedVsix Include="$(SolutionRoot)\src\Integration.Vsix\Manifests\VS201*\source.extension.vsixmanifest">
```

Comment:
The include pattern here is just the main VSIXes.
The `Find` pattern has been tweaked so it just updates the `Dependency` reference to the `EmbeddedVsix` project.